### DISCUSSION TITLE
a "joke" mode

### DISCUSSION THREAD

**Discussion by shinmorisawa** [ppy/osu#30258] [Ideas] [answered]
a joke mode option in the settings that make every text field seem like a joke. like "exit" would be "bye bye!!!!" and a "beatmap" would be a "circle clicky thingy"

**Comment by Tenexxt** [comment]
I think this could be pulled off by making it into a language (like Minecraft)

**Reply by shinmorisawa** [reply]
yeah true

**Comment by bdach** [comment, marked as answer]
i like to think that i'm not overly uptight in general but this proposal feels very "im so random *holds up spork*".
